### PR TITLE
[php] Add externs for `header_remove()`, `headers_list()` and `headers_sent()`

### PR DESCRIPTION
**Pull request by cedx**

### CODE REVIEW

File: std/php/Global.hx [hunk 1]
```diff
@@ -931,6 +931,21 @@ extern class Global {
...
 
+	/**
+		@see http://php.net/manual/en/function.header-remove.php
+	**/
+	static function header_remove(?name:String):Void;
+
+	/**
+		@see http://php.net/manual/en/function.headers-list.php
+	**/
+	static function headers_list():Array<String>;
```
Comment (RealyUniqueName) [inline std/php/Global.hx]: Return type should be `NativeIndexedArray<String>`

File: std/php/Global.hx [hunk 2]
```diff
@@ -931,6 +931,21 @@ extern class Global {
...
+
+	/**
+		@see http://php.net/manual/en/function.headers-sent.php
+	**/
+	static function headers_sent(?file:String, ?line:Int):Bool;
```
Comment (RealyUniqueName) [inline std/php/Global.hx]: arguments should be `?file:Ref<String>, ?line:Ref<Int>`

**Review by RealyUniqueName — CHANGES_REQUESTED** [review]

File: std/php/Global.hx (hunk 1 above)
Comment (CedX) [reply std/php/Global.hx]: Of course! :sweat_smile: Fixed.

File: std/php/Global.hx (hunk 2 above)
Comment (CedX) [reply std/php/Global.hx]: I was wondering how to specify arguments passed by reference. Thanks for the correction.

**Pull request merged**